### PR TITLE
Dockerize

### LINKED ISSUE #5: Migrate merritt ATOM feed code to python3 and deploy serverlessly
1) Rewrite for Python 3

2)  Add the additional identifier (ARK?) in the feed

- UCI indicated that not all of the Nuxeo-based "Local Identifiers" are being mapped through to Merritt's "local id" field. Requesting that we update the ATOM feed to include the local identifiers in additional repeated dc:identifier elements.
- Eric L. also noted that UCSF would like to see their local identifiers piped through to Merritt.
- Discussion thread w/ UC3 team: https://cdlib.slack.com/archives/CDT0H72BW/p1662594915311589
- UCI's notes: https://docs.google.com/document/d/1KbI3oaeud7fkf_CkzI0b4SZM7SrqIVtxC7orWuGKHIg/edit
- Nuxeo->Merritt ATOM feed specs: https://docs.google.com/document/d/1s0ijBWu7J2u_D9ZOudvi7a4hZIM1WKqDDdocuuYvEMg/edit#
- We’ll need to coordinate with Eric and Mark on updating Merritt’s code to detect and map through the additional identifiers.

3)  Get it running in Fargate

4) Regenerate ATOM feeds for all sources

5) Run ATOM feeds for newly setup sources. These 32 UCR collections need ATOM feeds:

https://registry.cdlib.org/admin/library_collection/collection/26757/
/asset-library/UCR/SCUA/Archival/WRCA041

https://registry.cdlib.org/admin/library_collection/collection/26758/
/asset-library/UCR/SCUA/Archival/WRCA056

https://registry.cdlib.org/admin/library_collection/collection/26759
/asset-library/UCR/SCUA/Archival/WRCA044

https://registry.cdlib.org/admin/library_collection/collection/27895
/asset-library/UCR/SpecialProjects/wrca_films

https://registry.cdlib.org/admin/library_collection/collection/28126/
/asset-library/UCR/SCUA/Archival/MS002

https://registry.cdlib.org/admin/library_collection/collection/26946
/asset-library/UCR/Test Space/DocNow

https://registry.cdlib.org/admin/library_collection/collection/26761/
/asset-library/UCR/GeneralLibraryCollection/BornDigitalMono

https://registry.cdlib.org/admin/library_collection/collection/27124
/asset-library/UCR/SpecialProjects/SIM

https://registry.cdlib.org/admin/library_collection/collection/28127
/asset-library/UCR/SCUA/Archival/UA310

https://registry.cdlib.org/admin/library_collection/collection/28128
/asset-library/UCR/SCUA/Archival/UA145

https://registry.cdlib.org/admin/library_collection/collection/28129
/asset-library/UCR/SCUA/Archival/UA082

https://registry.cdlib.org/admin/library_collection/collection/28130
/asset-library/UCR/SpecialProjects/WRCA270

https://registry.cdlib.org/admin/library_collection/collection/28131
/asset-library/UCR/SpecialProjects/klc

https://registry.cdlib.org/admin/library_collection/collection/28133
/asset-library/UCR/SCUA/Archival/MS054

https://registry.cdlib.org/admin/library_collection/collection/28134
/asset-library/UCR/SCUA/Archival/MS130

https://registry.cdlib.org/admin/library_collection/collection/28135
/asset-library/UCR/SCUA/Archival/MS233

https://registry.cdlib.org/admin/library_collection/collection/28136
/asset-library/UCR/SCUA/Archival/UA144

https://registry.cdlib.org/admin/library_collection/collection/28137
/asset-library/UCR/SCUA/Archival/MS171

https://registry.cdlib.org/admin/library_collection/collection/28138
/asset-library/UCR/SCUA/Archival/MS071

https://registry.cdlib.org/admin/library_collection/collection/28139
/asset-library/UCR/SCUA/Archival/MS139

https://registry.cdlib.org/admin/library_collection/collection/28140/
/asset-library/UCR/SCUA/Archival/MS170

https://registry.cdlib.org/admin/library_collection/collection/28141
/asset-library/UCR/SCUA/Archival/MS011

https://registry.cdlib.org/admin/library_collection/collection/28142
/asset-library/UCR/SCUA/Archival/MS205

https://registry.cdlib.org/admin/library_collection/collection/28143
/asset-library/UCR/SCUA/Archival/MS032

https://registry.cdlib.org/admin/library_collection/collection/28144
/asset-library/UCR/SCUA/Archival/MS065

https://registry.cdlib.org/admin/library_collection/collection/28145
/asset-library/UCR/SCUA/Archival/MS255

/asset-library/UCR/SCUA/Archival/UA127

### PR DESCRIPTION
**Pull request by barbarahui**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM public.ecr.aws/docker/library/python:3.9.19
```
Comment (bibliotechy) [inline Dockerfile]: Looking at https://gallery.ecr.aws/docker/library/python I see quite a few later python version 3.9.x

File: README.md [hunk 2]
```diff
@@ -1,10 +1,165 @@
...
+**sceptre/** - contains CloudFormation template for creating the AWS infrastructure needed to run the code in Fargate.
+
+**dags/** - contains the Airflow DAGs for the repo
+
+## Continuous Deployment
+
+When new code is pushed/merged into the main branch of this repo on github, 2 CodeBuild projects are triggered:
+
+1. `nuxeo_merritt-application-deploy` - builds the nuxeo merritt ATOM feed docker image and push to ECR
+2. `nuxeo_merritt-dags-deploy` - deploys DAGs and other code to Airflow
```
Comment (bibliotechy) [inline README.md:24]: What happens if one fails and one succeeds? Or if the dag deploy is faster than the docker image build?

File: Dockerfile (hunk 1 above)
Comment (barbarahui) [reply Dockerfile]: Oh, so there are! I think I weirdly assumed that the newest version was at the top...or something? I will try using a newer version!!

File: README.md (hunk 2 above)
Comment (barbarahui) [reply README.md:24]: If the application deploy fails and the DAG deploy succeeds, then the DAG will launch an older version of the ECS task. It's possible that the ECS task will fail if there is some incompatibility between the DAG code and the application code. If the ECS deploy succeeds and the DAG deploy fails, then we might have the same sort of incompatibility issue until the DAG deploy is fixed. Either way, we can roll back to an earlier commit pretty easily, I think. This is this exact deploy model we're using for Rikolti btw, so it's potentially an issue there too...

If the DAG deploy is faster than the docker image build, then I don't think it's a problem unless we kick off the DAG right away without waiting to see that the task deploy finished. Right now the DAG is configured to run weekly on Saturday, so I doubt we'll run into this, but I'll add a note to the README.

**Review by bibliotechy — DISMISSED** [review]

**Pull request merged**